api/api.go

```diff
@@ -698,9 +698,9 @@ func (c *Cluster) ToStorageCluster() *StorageCluster {
 		NodeId: c.NodeId,
 	}
 
-	cluster.Nodes = make([]*StorageNode, len(c.Nodes))
+	cluster.NodeIds = make([]string, len(c.Nodes))
```

Comment:
This is used only by the sdk? if so i'm ok with this change if not we can't do this.

Reply:
Yeah, the StorageCluster is an SDK-only structure.